src/Swarm/Language/Syntax.hs

```diff
@@ -100,14 +102,27 @@ import Swarm.Language.Types
 import Swarm.Util qualified as Util
 import Witch.From (from)
 
+-- | Maximum perception distance for
+-- 'chirp' and 'sniff' commands
+maxSniffRange :: Int32
+maxSniffRange = 200
```

Comment:
Most other constants in the game are powers of 2. Would it make sense to use either 128 or 256?  I don't think it's a big deal though.